### ISSUE TITLE
Import Records do not show on old database, but they show up on new database

### ISSUE THREAD

**Issue by nicholas512** [geocryology/sensorDb#12]

**Comment by nicholas512** [comment]
on a new test database, logs show up:
![image](https://user-images.githubusercontent.com/15223327/148617853-715056bc-ef47-4a58-936f-456e9cc8f7eb.png)

but on an old one they don't
![image](https://user-images.githubusercontent.com/15223327/148617888-6e2b6c87-cebd-45c6-9fb6-d937888d41e9.png)

**Comment by nicholas512** [comment]
page calls `https://pdsp.permafrostnet.ca/sensordb/db/getImportRecords?id=` which returns error 

```
{"cause":{"cause":{"error":"A JSONObject text must begin with '{' at 1 [character 2 line 1]"},"error":"Error retrieving import records from database"},"error":"Error retrieving all import records from database"}
```

**Comment by nicholas512** [comment]
It works if you change the SQL from
```
SELECT id,import_time,filename,import_parameters FROM imports 
```

to 
```
SELECT id,import_time,filename,import_parameters FROM imports LIMIT 2
```


Either there is data in some records that crashes, or there are too many records to return

**Comment by nicholas512** [comment]
resolved in #18
